src/api/app/controllers/source_project_config_controller.rb

```diff
@@ -0,0 +1,55 @@
+class SourceProjectConfigController < SourceController
```

Comment:
[nitpick] I would prefer that we follow the style of nesting the controllers in subfolders like we do e.g. with Kiwi.

```
source_project/configs_controller.rb
```

Reply:
I would prefer to do it later, specially because if we do it consequently we will have

```
source_project/
  - config_controller
  + package_project/
    - meta_controller.rb 
```

which will involve much more changes than simply moving code around.

Reply:
I'm fine with that, just wanted to point it out